extra/src/main/java/aima/extra/search/pqueue/uninformed/GraphPriorityQueueSearch.java

```diff
@@ -45,19 +46,19 @@
 		// node <- a node with STATE = problem.INITIAL-STATE
 		Node<A, S> node = newRootNode(problem.initialState());
 		// frontier <- a priority queue, with node as the only element
-		Queue<Node<A, S>> frontier = newFrontier(node);
-		// explored <- an empty set
+        PriorityQueue<Node<A, S>> pqFrontier = newFrontierPriority(node, getNodeComparator());
+        // explored <- an empty set

```

Comment:
There is no need to rename the variable from 'frontier' to 'pqFrontier' as it add no value and also makes it inconsistent with the pseudocode description.